recipe/meta.yaml

```diff
@@ -7,11 +7,12 @@ package:
 source:
   fn:  serf-{{ version }}.tar.bz2
   url: https://archive.apache.org/dist/serf/serf-{{ version }}.tar.bz2
-  sha256: 549c2d21c577a8a9c0450facb5cca809f26591f048e466552240947bdf7a87cc
+  sha256: be81ef08baa2516ecda76a77adf7def7bc3227eeb578b9a33b45f7b41dc064e6
   patches:
     - SConstruct.diff
 
 build:
+  skip: True # [s390x or aarch64]
```

Comment:
Im curious, why do we skip s390x and aarch64?

Reply:
I thought given usage of this tool and as we haven't supported it in the past, it wasn't worth the time at this point.

Reply:
Ok, maybe add a comment to say that `libapr` and `libaprutil` are not available on these platforms at the time of writing?